src/lib/dexrt.cpp

```diff
@@ -391,7 +391,13 @@ void dex_get_cuda_architecture(int device, char* arch) {
     printf("Invalid CUDA architecture version: %d.%d", majorVersion, minorVersion);
     std::abort();
   }
-  snprintf(arch, 5, "sm_%d%d", majorVersion, minorVersion);
+  if (majorVersion > 7 || (majorVersion == 7 && minorVersion > 5)) {
+    printf("CUDA architecture version (%d.%d) unsupported by LLVM 9, capping at 7.5",
```

Comment:
We probably don't want to print here, because it will keep repeatedly clobbering the output. I'd remove it or at least gate it behind a static variable to make sure that this only happens once.

Reply:
Sorry, missed this reply. Have removed the printf (it was indeed messing up the output)